examples/jsm/controls/OrbitControls.js

```diff
@@ -189,12 +189,13 @@ class OrbitControls extends EventDispatcher {
 
 				offset.copy( position ).sub( scope.target );
 
+				// so camera.up is the orbit axis
+				quat = new Quaternion().setFromUnitVectors( object.up, yAxisUp );
+				quatInverse = quat.clone().invert();
```

Comment:
You are instantiating 120 quaternions per second at 60 fps. Avoid `new` and `clone()` in such cases.

Reply:
Ok, done